app/uk/gov/hmrc/agentsubscription/model/subscriptionJourney/AmlsData.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentsubscription.model.subscriptionJourney
+
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+import play.api.libs.json._
+
+case class RegisteredDetails(membershipNumber: String, membershipExpiresOn: LocalDate)
```

Comment:
can we reuse the models from `uk/gov/hmrc/agentsubscription/model/AmlsDetails.scala` ?  or are these due to you just moved the package ?

Reply:
We are reorganising the data classes for the temporary store, and to ensure backward compatibility with the rest of the code we have tried to have completely separate classes for the temporary store.  Therefore I renamed this class to avoid confusion.  All the temporary store model classes are under model.subscriptionJourney (they are based on the originals, but will evolve separately).